pkg/apis/jaegertracing/v1/jaeger_types.go

```diff
@@ -152,6 +152,12 @@ type JaegerCommonSpec struct {
 	// +listType=set
 	Volumes []v1.Volume `json:"volumes,omitempty"`
 
+	// +optional
+	DNSPolicy v1.DNSPolicy `json:"dnsPolicy,omitempty"`
```

Comment:
Could you also add a PR to the documentation repo, operator's documentation, adding these new options under the "Finer grained configuration" section?

https://github.com/jaegertracing/documentation/blob/master/content/docs/next-release/operator.md#finer-grained-configuration